src/app/cake/cake.class.ts

```diff
@@ -0,0 +1,26 @@
+
+export enum CakeStatus {
```

Comment:
Since we are using a Single File Responsibility file distribution, this Interface should live in a separate file named `cake-status.enum.ts`.

Reply:
Done ✅